common/error.cpp

```diff
@@ -18,4 +18,25 @@ namespace mender::common::error {
 
 const Error NoError = Error(std::error_condition(), "");
 
+const CommonErrorCategoryClass CommonErrorCategory;
+
+Error MakeError(ErrorCode code, const std::string &msg) {
+	return Error(std::error_condition(code, CommonErrorCategory), msg);
+}
+
+const char *CommonErrorCategoryClass::name() const noexcept {
```

Comment:
Q: Why `const char*`

Reply:
This is an override, so it comes from the `std` interface. I don't know why they chose `const char *` instead of `const string &`, but since it's supposed to be unchanging, I guess it doesn't matter which one it is. :thinking: